_i18n/de/resources/moneropedia/remote-node.md

```diff
@@ -1,24 +1,24 @@
 ---
-terms: ["remote-node", "remote-nodes"]
-summary: "Nodes wich are not running in the same machine as the wallet. Open remote nodes allow to use the Monero network immediately"
+terms: ["remote-node", "remote-nodes", "Remote-Node", "Remote-Nodes"]
+summary: "Nodes, die nicht im gleichen Gerät wie das Wallet laufen. Offene Remote-Nodes erlauben die direkte Nutzung des Monero-Netzwerks"
```

Comment:
What about "auf demselben Gerät laufen", would that be okay? "Am selben Gerät" sounds odd to me in turn.

> I think "public remote nodes" is clearer and more commonly used than "open remote nodes", but I see they used "open" instead of "public" throughout the English text. When you want to make your node "public", you pass the parameter "--public-node" to monerod. Most other documentation also seems to use "public" instead of "open".

Asking @erciccione for input on this topic.

Reply:
"auf demselben" reads better, yes.

Reply:
"public" and "open" are used inconsistently across our tools and documentation. That's something that need to be fixed, but it's out of scope for this PR. The two word have different meaning though, "public" assumes the node is for public use (advertised with `--public-node`), open means that the node is accessible, but not necessarily advertised for public use. But yeah, it's a problem of consistency.